core/deployment/src/main/java/io/quarkus/deployment/configuration/RunTimeConfigurationGenerator.java

```diff
@@ -1630,4 +1640,16 @@ private static boolean isFieldEligibleForDirectAccess(ClassDefinition.ClassMembe
                 && Modifier.isPublic(classMember.getEnclosingDefinition().getConfigurationClass().getModifiers())
                 && Modifier.isPublic(classMember.getField().getType().getModifiers());
     }
+
+    private enum Type {
+        BuildTime("si"),
+        Bootstrap("bs"),
+        Runtime("rt");
```

Comment:
Uppercase them?

Reply:
Yeah I guess most of our enums are upper case, so I'll fix it

Reply:
Fixed